src/main/tut/index.html

```diff
@@ -515,6 +559,8 @@
     out += da.decode(s)
   }.result())
 ```
+<!-- COMMENT -->
+
```

Comment:
Yeah.... `CanBuildFrom` is such a tricky subject. I totally get where the motivation came from to add it here, as it is practically useful. With that being said, I feel like the mess `CanBuildFrom` brings along so much baggage (for example, the use here of `Nothing`, which requires explanation of bottom types and return of control to the caller). If it were me, i'd probably skip a generic implementation for all `CanBuildFrom` subtypes, and instead opt for specialized collection implementations where I can leverage the properties of that structure. For example, `Vector` vs `Stream`.